communication/slack-config/sig-docs/docs-channels.yaml

```diff
@@ -16,6 +16,7 @@ channels:
   - name: kubernetes-docs-pt
   - name: kubernetes-docs-ru
   - name: kubernetes-docs-ta
+  - name: kubernetes-docs-tw
```

Comment:
The language code for (traditional) Chinese is `zh`. We shouldn't make this change as-is.

Reply:
Hi @sftim , I understand that you're suggesting not to make the change as the language code "zh" yet it already exists for **Simplified Chinese**. Do you get suggested replacement name for the channel name?

Reply:
We have two options:

- use the existing channel
- rename the existing channel to `kubernetes-docs-zh-cn` and make a new channel `kubernetes-docs-zh-tw`

Even with the geopolitics involved, I would use the existing channel name and not make a new one. If that doesn't work out, we can consider a split.

Reply:
Got it. In that case, I will try to use the existing channel first. Thank you for your kind reply.